Assets/Plugins/Appboy/Editor/PostBuild.cs

```diff
@@ -1,10 +1,12 @@
-﻿using System;
+#if UNITY_IOS
+using System;
 using System.IO;
 using System.Text.RegularExpressions;
 using UnityEngine;
 using UnityEditor;
 using UnityEditor.Callbacks;
 using UnityEditor.iOS.Xcode;
```

Comment:
why gate all dependencies instead of just `using UnityEditor.iOS.Xcode;`?

Reply:
Because none of them are used unless you are on iOS. Either way is fine though.

Reply:
works for me